modules/ROOT/partials/integrations/jquery-quick-start.adoc

```diff
@@ -0,0 +1,191 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <script src="/public/jquery/jquery.min.js"></script>
+    <script src="/public/tinymce/tinymce.min.js" referrerpolicy="origin"></script>
+    <script src="/public/tinymce-jquery.min.js"></script>
+  </head>
+  <body>
+    <div>
+      <textarea id="tiny">&lt;p&gt;Welcome to the TinyMCE jQuery example!&lt;/p&gt;</textarea>
+    </div>
+    <script>
+      $('textarea#tiny').tinymce({
+        height: 500,
+        menubar: false,
+        plugins: [
+          'advlist', 'autolink', 'lists', 'link', 'image', 'charmap', 'preview',
+          'anchor', 'searchreplace', 'visualblocks', 'code', 'fullscreen',
+          'insertdatetime', 'media', 'table', 'code', 'help', 'wordcount'
+        ],
+        toolbar: 'undo redo | blocks | bold italic backcolor | ' +
+          'alignleft aligncenter alignright alignjustify | ' +
+          'bullist numlist outdent indent | removeformat | help'
+      });
+    </script>
+  </body>
+</html>
+----
+endif::[]
+ifeval::["{productSource}" == "cloud"]
+[source,html,subs="attributes+"]
+----
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <script
+      src="https://code.jquery.com/jquery-3.6.0.min.js"
+      integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4="
+      crossorigin="anonymous"
+    ></script>
+    <script
+      src="{cdnurl}"
+      referrerpolicy="origin"
+    ></script>
+    <script src="https://cdn.jsdelivr.net/npm/@tinymce/tinymce-jquery@1/dist/tinymce-jquery.min.js"></script>
+  </head>
+  <body>
+    <div>
+      <textarea id="tiny">&lt;p&gt;Welcome to the TinyMCE jQuery example!&lt;/p&gt;</textarea>
+    </div>
+    <script>
+      $('textarea#tiny').tinymce({
+        height: 500,
+        menubar: false,
+        plugins: [
+          'advlist', 'autolink', 'lists', 'link', 'image', 'charmap', 'preview',
+          'anchor', 'searchreplace', 'visualblocks', 'code', 'fullscreen',
+          'insertdatetime', 'media', 'table', 'code', 'help', 'wordcount'
+        ],
+        toolbar: 'undo redo | blocks | bold italic backcolor | ' +
+          'alignleft aligncenter alignright alignjustify | ' +
+          'bullist numlist outdent indent | removeformat | help'
+      });
+    </script>
+  </body>
+</html>
+----
+endif::[]
+== TinyMCE in a jQuery UI Dialog
+To render {productname} instances inside jQuery UI dialogs, add the following code:
+[source,js]
+----
+// Prevent jQuery UI dialog from blocking focusin
+$(document).on('focusin', function(e) {
+  if ($(e.target).closest(".tox-tinymce, .tox-tinymce-aux, .moxman-window, .tam-assetmanager-root").length) {
+    e.stopImmediatePropagation();
+  }
+});
+----
+
+This code is required because jQuery blocks all `focusin` calls from elements outside the dialog. For a working example, http://fiddle.tiny.cloud/rsdaab/840[try this {productname} fiddle].
```

Comment:
Looks like this fiddle needs updating, as it's still using `jquery.tinymce.min.js`.

Reply:
Created task INT-2833 to fix this.